src/packages/carpool-view-flow/react/screens/rides-list-with-data/RidesListWithDataScreen.jsx

```diff
@@ -0,0 +1,37 @@
+import { createContainer } from 'meteor/react-meteor-data';
+import RidesList from '../../components/rides-list/RidesList';
+
+/*global Progress*/
+/*global carpoolService*/
+
+export default createContainer(({filterOwn = "all", role = "driver", aLoc, bLoc, bTime}) => {
```

Comment:
Why this appears in screens/ 
Screens suppose to be agnostic view rendering components.